generic_expression_patterns_modules/make_GiG_network.R

```diff
@@ -0,0 +1,41 @@
+# Using R 3.6.3
+
+# Manually load operons.rda and geneinfo.rda into R environment
+# from https://github.com/greenelab/ADAGEpath/blob/master/data/
+if (!requireNamespace("BiocManager", quietly = TRUE))
+  install.packages("BiocManager")
+BiocManager::install("impute")

```

Comment:
Do these need to be installed every time you run the script? Maybe you could check if they're installed before installing them. Something like this might work: https://stackoverflow.com/a/4090208